client/layout/sidebar/style.scss

```diff
@@ -281,6 +281,16 @@ form.sidebar__button {
 			}
 		}
 
+		form.sidebar__button {
```

Comment:
Can't these added `form.sidebar__button` rule be "merged" in some way with the `a` rule (just above) so we don't have to remember to edit twice a styling that does the same thing?